### PR TITLE
Automated PR for NCSA

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/NCSA.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  </urls>
+  <license>
+    <title>
+      <p>University of Illinois/NCSA Open Source License</p>
+    </title>
+    <copyright>
+      <p>Copyright (c) 
+        <alt name="copyright" match=".+"> &lt;Year&gt; &lt;Owner Organization Name&gt;</alt>. All rights reserved. 
+      </p>
+      <p>Developed by:
```
Comment (jlovejoy) [inline src/NCSA.xml]: should this be part of copyright tag?  (also may want to check if LLVM uses this line - if not, may want to make this line optional)

Comment (bradleeedmondson) [inline src/NCSA.xml]: agree; i'd separate the developed by into a "note" tag

Comment (pmadick) [inline src/NCSA.xml]: Agree.

Comment (jlovejoy) [inline src/NCSA.xml]: move end copyright tag up, discussed on 8/16 call

**Pull request merged**